src/main/target/KAKUTEF7/target.h

```diff
@@ -22,8 +22,17 @@
 
 //#define USE_TARGET_CONFIG
 
+#ifdef KAKUTEF765
```

Comment:
Can we please have this as 2 separate pull request, one for the F765 and one for the new target?

Reply:
It quite makes sense to me in having the first ever F765 target added in the same PR as MCU support, but I don't care.